### ISSUE TITLE
lambda execution error

### ISSUE THREAD

**Issue by xsnrg** [neillturner/terraform-aws-alb-logs-to-elasticsearch#5]
The following stack trace is generated when the lambda tries to run:

```
errorMessage | Error: Cannot find module 'exports'Require stack:- /var/runtime/UserFunction.js- /var/runtime/index.js
-- | --
errorType | Runtime.ImportModuleError
stack.0 | Runtime.ImportModuleError: Error: Cannot find module 'exports'
stack.1 | Require stack:
stack.2 | - /var/runtime/UserFunction.js
stack.3 | - /var/runtime/index.js
stack.4 | at _loadUserApp (/var/runtime/UserFunction.js:100:13)
stack.5 | at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)
stack.6 | at Object.<anonymous> (/var/runtime/index.js:43:30)
stack.7 | at Module._compile (internal/modules/cjs/loader.js:999:30)
stack.8 | at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
stack.9 | at Module.load (internal/modules/cjs/loader.js:863:32)
stack.10 | at Function.Module._load (internal/modules/cjs/loader.js:708:14)
stack.11 | at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
stack.12 | at internal/main/run_main_module.js:17:47

```

**Comment by neillturner** [comment]
code hasn't changed so only thinking is this is related to the nodejs release. Updated the dependencies to 16.x.  and released new version 0.2.2. If you could test that would be great.

**Comment by baptiste-s** [comment]
Hi Neil, 

I am got the same error. I don't understand, how are we supposed to update to 16.x, Lambda only support 14.x. Correct me if I'm wrong. 

anyway, to make it work I had to override the tf variable nodejs_version to 14.x when deploying. Then I had to manually change the handler in lambda from exports.handler to index.handlers.

hope that help.

**Comment by xsnrg** [comment]
@baptiste-s with your change to index.handlers and node to 14, did the whole lambda then work ?

**Comment by baptiste-s** [comment]
> @baptiste-s with your change to index.handlers and node to 14, did the whole lambda then work ?

yes it does work

**Comment by xsnrg** [comment]
any chance you could  put a PR up for @neillturner to merge?

**Comment by xsnrg** [comment]
I have updated a PR that should fix the open issues. It includes:
Fixes for nodejs versions
removing some defaults as they were not working correctly
passing `region` through to the lambda as it was pretty much hard coded in the index.js 
documentation updates

The logging lambda is working for me properly with this branch/PR

**Comment by xsnrg** [comment]
Closed by #6
